src/documents/IoCTestingFramework.md

```diff
@@ -333,25 +577,60 @@ fluid.defaults("fluid.tests.asyncTestTree", {
 });
 ```
 
+### `markupFixture` property supporting fixtures written against markup in the host document
```

Comment:
The flow here seems a bit strange to me as a reader. You show me part of the code example, and then go back into description of the features of the system, and then later dive back into the example again. I ended up getting confused, searching for reference to the <code> fluid.tests.asyncTester</code> in the document, and then coming back to read this material later.

Perhaps we should move the rest of the example, which is down starting at line 642, up here, and then reserve the remaining material for the end of the document?